docs/docs/cloud/deployment/test_locally.md

```diff
@@ -0,0 +1,89 @@
+# How to test a LangGraph app locally
+
+This guide assumes you have a LangGraph app correctly set up with a proper configuration file and a corresponding compiled graph, and that you have a proper LangChain API key.
+
+## Setup
+
+Install the proper packages:
+
+```shell
+pip install langgraph-cli
+```
+
+## Start the API server
+
+Once you have downloaded the CLI, you can run the following command to start the API server for local testing:
+
+```shell
+langgraph up
+```
+
+This will start up the LangGraph API server locally. If this runs successfully, you should see something like:
+
+```shell
+Ready!
+- API: http://localhost:8123
+2024-06-26 19:20:41,056:INFO:uvicorn.access 127.0.0.1:44138 - "GET /ok HTTP/1.1" 200
+```
+
+### Interact with the server
+
+We can now interact with the API server using the LangGraph SDK. First, we need to start our client, select our assistant (in this case a graph we called "agent", make sure to select the proper assistant you wish to test).
+
+=== "Python"
+
+    ```python
+    from langgraph_sdk import get_client
+
+    client = get_client(url="whatever-your-local-deployment-url-is")
```

Comment:
wont this have a default value? can we just put it here with a comment saying that you may need to change?